Content.Shared/_starcup/Roles/JobPrototype.JobSpawns.cs

```diff
@@ -0,0 +1,11 @@
+namespace Content.Shared.Roles;
+
+public sealed partial class JobPrototype
+{
+    /// <summary>
+    /// starcup: If a job should always spawn at its job spawn rather than late join spawn points or arrivals.
+    /// Used for roles like prisoner that always need to spawn in specific locations.
+    /// </summary>
+    [DataField("alwaysUseJobSpawn")]
```

Comment:
```suggestion
    [DataField]
```

DataField infers camelCase naming from the variable name now. Check out https://hackmd.io/@Slart/S1hsoGFm1l?stext=19328%3A65%3A0%3A1752560939%3A9F7rTw

Reply:
This is another case of my preference for being explicit.